tracer/test/Datadog.Trace.Tests/Debugger/DebuggerSettingsTests.cs

```diff
@@ -175,6 +185,19 @@ public void CodeOriginEnabled_False(string value)
                 settings.CodeOriginForSpansEnabled.Should().BeFalse();
             }
 
+            [Theory]
+            [InlineData("")]
+            [InlineData(null)]
+            [InlineData("2")]
+            public void CodeOriginEnabled_Null(string value)
```

Comment:
Same here, calling could shouldn't be deciding whether null means true or false. Plus we should be recording the "default" value in telemetry, which this currently isn't really doing